features/guestorder.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Guest order Place
+
+
+Background:Guest Order Place
+    Given The guest user will navigates to homepageurl
+    And Guest user will clicks on the respective Product
+
+    Scenario: Guest Order Place (Negative scenario)
+    Given The guest user can be navigated to product details page 
```

Comment:
@karthikeyan7010 I already told you not to create big scenarios and try to split into multiple scenarios